### PR TITLE
Removed advertisement-interface and capture length

### PR DESCRIPTION
**Pull request by MichaelBaj**

### CODE REVIEW

**Review by Chr1st0ph3rTurn3r — DISMISSED** [review]

**Review by avinashbhat80 — APPROVED** [review]

**Review by ayungelson — APPROVED** [review]

**Review by Joe-Swetz — APPROVED** [review]

File: docs/config_ha_vrrp.md [hunk 1]
```diff
@@ -90,20 +85,11 @@ Activate VRRP on the `wan` and `lan` device interfaces of node1. By configuring
                     name               wan
                     type               ethernet
                     pci-address        0000:00:05.0
-                    capture-filter     len>0
-
-                    vrrp
```
Comment (avinashbhat80) [inline docs/config_ha_vrrp.md:95]: Is this intentional? Don't we want to show an example for `Wan` side redundancy too?

Comment (Joe-Swetz) [reply docs/config_ha_vrrp.md:95]: I believe we allow configuring VRRP on the wan side, but I'm not sure if this is a common use case?

Comment (MichaelBaj) [reply docs/config_ha_vrrp.md:95]: VRRP requires a switch in front of it, correct? It is atypical for WAN interfaces to be configured with VRRP in production deployments. While technically possible, I would prefer our examples to follow best practices. I know of one instance where VRRP was incorrectly provisioned on the WAN without supporting infrastructure.
I do value your input however. Let me know if I am making a mistake by removing this example.

Comment (avinashbhat80) [reply docs/config_ha_vrrp.md:95]: I understand, the audience here is different and it makes sense. It is a well tested functionality and may be this is not a right place to advertise.  Just want to make sure the info is available some where for those who may be interested in wan side support.

**Review by avinashbhat80 — DISMISSED** [review]

**Review by Chr1st0ph3rTurn3r — DISMISSED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by Joe-Swetz — APPROVED** [review]

**Pull request merged**